### PR TITLE
Add react-google-analytics to quilt

### PR DESCRIPTION
**Pull request by antoinegrant**
Resolves https://github.com/Shopify/quilt/issues/284

### CODE REVIEW

File: packages/react-google-analytics/src/GaJS.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  Props,
+  never
+> {
+  render() {
+    const {account, debug} = this.props;
+
+    return (
+      <>
+        <script
+          type="text/javascript"
```
Comment (kaelig) [inline packages/react-google-analytics/src/GaJS.tsx]: You can safely delete this line

File: packages/react-google-analytics/src/Universal.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  never
+> {
+  render() {
+    const {debug} = this.props;
+
+    return (
+      <>
+        <script
+          id="google-analytics-universal-script"
+          type="text/javascript"
```
Comment (kaelig) [inline packages/react-google-analytics/src/Universal.tsx]: This line can also safely be removed.

**Comment by kaelig** [comment]
Did you mean to leave the README empty?

**Comment by antoinegrant** [comment]
🏓 for another round of reviews. 🙏

**Comment by antoinegrant** [comment]
Comments addressed! 👍

File: packages/react-google-analytics/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```jsx
+import {
+  Universal,
+  GaJS,
+} from '@shopify/react-google-analytics';
+
+const UNIVERSAL_GA_ACCOUNT_ID = 'UA-xxxx-xx';
+const GA_JS_ACCOUNT_ID = 'UA-xxxx-xx';
+
+<Universal
```
Comment (kaelig) [inline packages/react-google-analytics/README.md]: This is unclear - would one ever use both GaJS and Universal at the same time in an app? Or should users only import one that fits their need?

Which one is the recommended way?

Comment (antoinegrant) [reply packages/react-google-analytics/README.md]: Good point, I guess I was focused on how we use it. We have to load both scripts in order to be on par with the way we do analytics in core.

I'll update the docs to just use or the other.

Thanks

**Comment by antoinegrant** [comment]
@kaelig I've made some changes to the docs.

**Comment by antoinegrant** [comment]
@kaelig I've added a little info https://github.com/Shopify/quilt/pull/285/commits/1ebe4e2e8e7f070cde36677e8b871e21ab64df34

**Comment by antoinegrant** [comment]
Merging this PR. @TheMallen are you able to run the release script?

**Pull request merged**